src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/rc_get_interior_vehicle_data_consent_test.cc

```diff
@@ -267,11 +267,14 @@ TEST_F(RCGetInteriorVehicleDataConsentTest,
        Run_MobileSendButtonPressMessage_HMISendINUSEModeToMobile) {
   // Arrange
   auto mobile_message = CreateBasicMessage();
+  auto msg_ver = utils::SemanticVersion();
 
   // Expectations
   EXPECT_CALL(mock_allocation_manager_, AcquireResource(_, _, _))
       .WillOnce(Return(rc_rpc_plugin::AcquireResult::IN_USE));
 
+  EXPECT_CALL(*mock_app_, msg_version()).WillOnce(ReturnRef(msg_ver));
```

Comment:
@YarikMamykin if we are not testing SemanticVersion here, please use OnCall

Reply:
@alexkutsan , fixed in 504844c7e81194da100c5a113c4f016acfb34b50